orc8r/cloud/docker/controller/supervisor_logger.py

```diff
@@ -42,18 +37,14 @@ def main():
         data = sys.stdin.read(int(headers['len']))
 
         # transition from READY to ACKNOWLEDGED
-        write_stdout('RESULT %s\n%s' % (len(data), data))
-
+        write_stdout('RESULT 2\nOK')
```

Comment:
Why is a constant value returned here? Do we expect that the loop is ended before if `2!=len(data)` or 'data!=OK'?